rca/research/models.py

```diff
@@ -0,0 +1,6 @@
+from rca.utils.models import BasePage
+
+
+class ResearchCentrePage(BasePage):
+    template = "patterns/pages/researchcentre/research_centre.html"
+    pass
```

Comment:
Technically this doesn't need to be here... it's about to change in another pr though